assets/findify-pagination.css

```diff
@@ -5,6 +5,7 @@
 
 .findify-lazy-loading-pagination-container {
   display: flex;
+  flex-direction: column-reverse;
```

Comment:
What this would solve?
row-reverse and column-reverse are very specific cases. they should not be used light minded.
Wouldn't it be better to switch position on layout itself? Let me understand this.

Reply:
it is easiest way to achive desired result.  https://i.imgur.com/by4ALP5.png

Reply:
btw Who added this??